changelog.md

```diff
@@ -1,4 +1,6 @@
 # Future
+- [FIXED] sql syntax issue due to missing space in Postgresql query interface function "dropFunction" 
```

Comment:
Lets keep it simple, Like

``[FIXED] `dropFunction` syntax errors (PostgresSQL)``